[pull] main from kubeflow:main

dhirajsb opened the pull request:
See [Commits](/opendatahub-io/model-registry/pull/312/commits) and [Changes](/opendatahub-io/model-registry/pull/312/files) for more details.

- keep https://github.com/kubeflow/model-registry/pull/1531
- keep https://github.com/kubeflow/model-registry/pull/1520
- keep https://github.com/kubeflow/model-registry/pull/1528
- keep https://github.com/kubeflow/model-registry/pull/1505

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- New Features
  - Added Model Catalog details page with breadcrumb, provider info, and description.
  - Cards now navigate to detail views; routing consolidated under /model-catalog/*.
  - Supports model IDs with special characters in URLs.

- Tests
  - Python tests use auth tokens and configurable SSL.
  - Added Cypress E2E for details page and improved page-object helpers.

- Documentation
  - Updated license scan badge link.

- Chores
  - Removed obsolete license-scanning workflow.
  - Bumped a test dependency.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

tarilabs reviewed (approved):
thanks for the headsup

tarilabs committed:
chore: update FOSSA to new FOSSA project ID (#1531)

Description

follow-up to:
- https://github.com/kubeflow/model-registry/issues/323

the Kubeflow Community has decided to opt for OAuth setup, instead of what originally acted on with https://github.com/kubeflow/model-registry/issues/323#issue-2496435220 with a push-only token safely exposed per recommendation:
- https://github.com/fossa-contrib/fossa-action?tab=readme-ov-file#push-only-api-token

and also as adopted by other CNCF projects:
- https://github.com/cncf/foundation/issues/109

either way, aligning with Kubeflow community decision with this PR.

How Has This Been Tested?

https://github.com/kubeflow/model-registry/pull/1527#event-19364561505

Signed-off-by: Matteo Mortari <matteo.mortari@gmail.com>

dependabot[bot] committed:
build(deps): bump github.com/stretchr/testify from 1.10.0 to 1.11.0 in /clients/ui/bff (#1520)

Bumps [github.com/stretchr/testify](https://github.com/stretchr/testify) from 1.10.0 to 1.11.0.
- [Release notes](https://github.com/stretchr/testify/releases)
- [Commits](https://github.com/stretchr/testify/compare/v1.10.0...v1.11.0)

---
updated-dependencies:
- dependency-name: github.com/stretchr/testify
  dependency-version: 1.11.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dbasunag committed:
Add support for setting verify ssl, user token and request headers in tests (#1528)

* Add support for setting verify ssl, user token and request headers in tests

Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

* Update clients/python/tests/conftest.py

Co-authored-by: Sidney Glinton <sglinton@redhat.com>
Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

* Update clients/python/tests/conftest.py

Co-authored-by: Sidney Glinton <sglinton@redhat.com>
Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

* Update clients/python/tests/REST_bindings_test.py

Co-authored-by: Sidney Glinton <sglinton@redhat.com>
Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

* Update clients/python/tests/REST_bindings_test.py

Co-authored-by: Sidney Glinton <sglinton@redhat.com>
Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>

---------

Signed-off-by: Debarati Basu-Nag <dbasunag@redhat.com>
Co-authored-by: Sidney Glinton <sglinton@redhat.com>

YuliaKrimerman committed:
Model Catalog Details Page (#1505)

Signed-off-by: Yulia Krimerman <juliapiterova@hotmail.com>

dhirajsb committed:
Merge remote-tracking branch 'origin/main' into chore/dbokde-20250827-sync

Signed-off-by: Dhiraj Bokde <dhirajsb@users.noreply.github.com>

dhirajsb commented:
Includes the following kubeflow PRs

- keep https://github.com/kubeflow/model-registry/pull/1531
- keep https://github.com/kubeflow/model-registry/pull/1520
- keep https://github.com/kubeflow/model-registry/pull/1528
- keep https://github.com/kubeflow/model-registry/pull/1505

/lgtm
/approve
